src/main/java/com/school/project/dto/LessonDto.java

```diff
@@ -0,0 +1,22 @@
+package com.school.project.dto;
+
+import com.school.project.model.entities.Subject;
+import com.school.project.web.GroupWeb;
+import com.school.project.web.UserWeb;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class LessonDto {
```

Comment:
@MichelRogov is this dto complete enough to be handled by the front end and be rendered? Dont we need to return full thema, group and teacher objects instead of just ids? If we return only ids in dto, front end will need to make more additional calls to backend to render the dto.

Reply:
@sergeylukichev  at the last meeting we discussed this question, and now we are changing the DTO, so that it returns not only Id, but the fullness of the Object(DTO)

Reply:
Yeah, it supposed to be done later, right now we even don't have all the dto's in model since everybody is still working on his part.